### PR TITLE
Fixing Issue #571,#582 and #572

### PR DESCRIPTION
**Pull request by rishubhjain**

### CODE REVIEW

File: tendrl/commons/message.py [hunk 1]
```diff
@@ -141,9 +141,8 @@ def __init__(self, priority, publisher, payload):
                 err = "Exception field is not found in payload"
                 sys.stderr.write(err)
         else:
-            err = "Given exception %s is not a subclass of " \
-                "Exception class \n" % (str(payload["exception"]))
-            sys.stderr.write(err)
+            err = "Given exception  is not a subclass of " \
+                "Exception class \n" % sys.stderr.write(err)
```
Comment (r0h4n) [inline tendrl/commons/message.py]: where is the string formatter in above string?

File: tendrl/commons/objects/node/atoms/is_node_tendrl_managed/__init__.py [hunk 2]
```diff
@@ -9,57 +9,59 @@ def __init__(self, *args, **kwargs):
         super(IsNodeTendrlManaged, self).__init__(*args, **kwargs)
 
     def run(self):
-        node_ids = self.parameters.get('Node[]')
-        if not node_ids or len(node_ids) == 0:
-            raise AtomExecutionFailedError("Node[] cannot be empty")
+        if self.parameters:
```
Comment (r0h4n) [inline tendrl/commons/objects/node/atoms/is_node_tendrl_managed/__init__.py]: why is this file changed?

Comment (rishubhjain) [inline tendrl/commons/objects/node/atoms/is_node_tendrl_managed/__init__.py]: @r0h4n It was related to https://github.com/Tendrl/commons/issues/572, I felt that there must be a check as self.parameters could be None

Comment (r0h4n) [inline tendrl/commons/objects/node/atoms/is_node_tendrl_managed/__init__.py]: I have handled it in the baseflow, baseatom classes, please remove this change,

more details https://github.com/Tendrl/commons/issues/572

Comment (rishubhjain) [inline tendrl/commons/objects/node/atoms/is_node_tendrl_managed/__init__.py]: @r0h4n Done,
